mandatory/1-fix-functions.js

```diff
@@ -31,12 +31,13 @@ function greaterThan10(num) {
 }
 
 function get3rdIndex(arr) {
```

Comment:
Correct me if I'm wrong but if its the third index does index not need to be set to 2 not 3?

Reply:
Hi Matt, in this example, they want 4th of element of array so index is number3 ...